custom/icds_reports/management/commands/sql_scripts/update_child_health_monthly.sql

```diff
@@ -0,0 +1,27 @@
+CREATE OR REPLACE FUNCTION update_child_health_monthly(date) RETURNS VOID AS
+$BODY$
+DECLARE
+  _tablename text;
+  _ucr_child_monthly_table text;
+  _start_date date;
+  _ucr_child_health_cases_table text;
+BEGIN
+  _start_date = date_trunc('MONTH', $1)::DATE;
+  _tablename := 'child_health_monthly' || '_' || _start_date;
+  EXECUTE 'SELECT table_name FROM ucr_table_name_mapping WHERE table_type = ' || quote_literal('child_health_monthly') INTO _ucr_child_monthly_table;
+  EXECUTE 'SELECT table_name FROM ucr_table_name_mapping WHERE table_type = ' || quote_literal('child_list') INTO _ucr_child_health_cases_table;
+
+  EXECUTE 'UPDATE ' || quote_ident(_tablename) || ' chm_monthly SET ' ||
+      'dob = child_health_monthly.dob, ' ||
```

Comment:
It should be be possible to pull `dob` and `sex` from teh child health cases query below

Reply:
Could you also pull `disabled`, `resident`, `minority`, and `caste` from the child health cases table as well? They were added in the smae migration, but I'm not sure if they were ever fully migrated

Reply:
@emord updated, I noticed in aggregation query that the ```dob``` and ```sex``` we take from ```child_health_monthly``` so I wanted to use the same.